lib/street_address.rb

```diff
@@ -880,7 +880,7 @@ def line1(s = "")
           s += " " + suffix unless suffix.nil?
         end
 
-        return s
+        s.strip
```

Comment:
@jbielick ☝️  force pushed this-- that should deal with the leading space in those edge cases where there's no number